.github/workflows/release.yml

```diff
@@ -12,6 +12,8 @@ on:
       - 'VERSION'
       - 'bazel/**'
       - 'pkg/**'
+  release:
+    types: []
```

Comment:
We can remove this since, as you point out, the only jobs triggered are via `feature.yml`.